README.md

```diff
@@ -269,7 +269,7 @@ ESP8266 problems can be fiddly to troubleshoot. Try the suggestions here if you'
 If you see errors like "Failed to connect to ESP8266" then your ESP8266 is probably not entering the bootloader properly:
 
 * Check you are passing the correct serial port on the command line.
-* Check you have permissions to access the serial port, and other software (such as modem-manager on Linux) is not trying to interact with it.
+* Check you have permissions to access the serial port, and other software (such as modem-manager on Linux) is not trying to interact with it. A common pitfall is leaving a serial terminal accessing this port open in another window and forgetting about it.
```

Comment:
@jrobeson : You're welcome to rewrite it like you think will be better, while keeping it coherent, and submit new patch, I'll close this one then.